content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2248,19 +2304,26 @@ The _name_ of the `fields` argument on the `@@index` attribute can be omitted:
 @@index([title, author])
 ```
 
```

Comment:
same as for @@unique, these go into the field reference

Reply:
I've added an example snippet to demonstrate this - is this correct?

Reply:
yes.